src/lib/features/project/project-service.ts

```diff
@@ -304,6 +311,18 @@ export default class ProjectService {
             await this.validateEnvironmentsExist(environments);
         }
     }
+
+    async validateProjectLimit() {
+        if (!this.flagResolver.isEnabled('resourceLimits')) return;
+
+        const limit = Math.max(this.resourceLimits.projects, 1);
```

Comment:
Any particular reason why you wanna put this minimum here and not in the parsing of the value?

Reply:
Will do this in the next PR. I like your idea more.